numba/tests/test_ir_inlining.py

```diff
@@ -732,7 +732,7 @@ def impl(x):
         def impl():
             a = bar(1)  # integer literal, should inline
             b = bar(2.3)  # float literal, should not inline
-            c = bar(3j)  # complex literal, should inline by virtue of cost model
+            c = bar(3j)  # complex literal, should inline via cost model
```

Comment:
I think this change is incorrect, it is by virtue of the cost model's decision that the inlining takes place, the cost model does not take part in the inlining itself.